node-services/cards-reminder/src/cardsReminder.ts

```diff
@@ -88,42 +92,64 @@ const cardsReminderService = new CardsReminderService(
 );
 
 app.get('/status', (req, res) => {
-    authorizationService.isAdminUser(req).then((isAdmin) => {
-        if (!isAdmin) res.status(403).send();
-        else res.send(cardsReminderService.isActive());
-    });
+    authorizationService
+        .isAdminUser(req)
+        .then((isAdmin) => {
+            if (!isAdmin) res.status(403).send();
+            else res.send(cardsReminderService.isActive());
+        })
+        .catch((err) => {
+            logger.error('Error getting authorization in GET /status' + err);
```

Comment:
The error could non be in authorization, so the error message should just be " Error in GET /status" and should return a an error status code (500?)